app/src/main/java/in/antaragni/ant/MainActivity.java

```diff
@@ -61,13 +83,16 @@ protected void onCreate(Bundle savedInstanceState)
       .withTranslucentStatusBar(false)
       .withActionBarDrawerToggleAnimated(true)
       .addDrawerItems(
-        new PrimaryDrawerItem().withName(R.string.drawer_item_home).withIcon(FontAwesome.Icon.faw_home).withIdentifier(HOME),
```

Comment:
why add new white space? if it was added by reformatting then fine. But if u have added it manually, remove them